contracts/ClaimsManager.sol

```diff
@@ -508,36 +508,31 @@ contract ClaimsManager is
         }
     }
 
-    function timeOutClaim(uint256 claimIndex) external override {
+    function claimIsTimedOut(uint256 claimIndex)
```

Comment:
I'm not sure where we would use this `claimIsTimedOut()` function, as the frontend will show a deadline to the user (that counts down in real time).

Reply:
I wanted to keep this for documentation, not necessarily for the frontend to use. Removing it altogether may be the better idea, I'll think about it.